CONTRIBUTING.md

```diff
@@ -32,12 +32,6 @@ You can then run the build using:
 yarn build
 ```
 
-Or run just the TypeScript build for type checks:
-
-```sh
-yarn build:types
```

Comment:
This is indeed outdated but the command is now "yarn run check". Can you update this if you don't mind please? :)